src/test/java/SetCollectionTest.java

```diff
@@ -0,0 +1,65 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+public class SetCollectionTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        //given
+        numbers = new HashSet<>();
+
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Nested
+    class 요구_사항_1 {
+        @Test
+        void set_size_확인() {
+            //when
+            int size = numbers.size();
+
+            //then
+            assertThat(size).isEqualTo(3);
+        }
```

Comment:
```suggestion
        assertThat(size).hasSize(3);
```

collection의 경우 위와 같이 구현도 가능함
위와 같이 한 라인으로 구현하는 것은 어떨까?